src/components/rating/readme.md

```diff
@@ -0,0 +1,68 @@
+# gx-rating
+
+The `gx-rating` allows displaying scores or rate items using stars.
+
+- `gx-rating` with `readonly` disabled or false:
```

Comment:
```suggestion
## Samples

- `gx-rating` with `readonly` disabled or false:
```

Reply:
That's true... I forgot include a title like _Usage Example_ or something like that. 😅

Reply:
Ready!